android-lib/src/androidTest/java/com/kinvey/androidTest/store/DeltaCacheTest.java

```diff
@@ -149,7 +151,8 @@ public void testReadALotOfItems() throws InterruptedException, IOException {
         testManager.createPersons(store, TEN_ITEMS);
 
         store.pushBlocking();
-        List<Person> pulledPersons = store.pullBlocking(client.query()).getResult();
+        Query query = client.query().addSort("_kmd", AbstractQuery.SortOrder.ASC);
```

Comment:
Should the sorting here be done by `_kmd.ect` rather than just `_kmd`?

Reply:
In the test doesn't matter what we set in sort parameters, there is important that query has any sorting.